source/_integrations/knocki.markdown

```diff
@@ -16,12 +16,48 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-The **Knocki** {% term integration %} can receive events for the triggers set up from your [Knocki devices](https://knocki.com/).
+The **Knocki** {% term integration %} allows you to trigger your favorite automations simply by tapping custom patterns (such as triple taps) on ordinary surfaces.
+
+## What is Knocki?
+
+[Knocki](http://knocki.com) is a small device that transforms common surfaces (including walls, tables, countertops, furniture, doors, and more) into touch control interfaces. You can configure specific tap patterns (such as a triple-tap, or two double-taps) to trigger your favorite Home Assistant automations!
+
+Unlike voice control, Knocki has no microphone, so it never listens in.  Plus, it allows your favorite functions to be within reach, without having to access a phone screen, button, or switch.
 
 ## Prerequisites
 
-To use the Knocki integration, you need a Knocki account.
+Make sure you have a [Knocki device](http://knocki.com) linked to your Knocki account before you set up this integration.
+If you don’t have a Knocki account, it’s fast and easy to set one up using the free Knocki mobile app.
+
+## Step-by-Step Instructions
+
+Start the setup process in the knocki mobile app (steps 1-7), and complete the process in Home Assistant (steps 8-13). **Steps 8-10 cannot be completed prior to steps 1-7.**
+
+### Steps 1-7:  Start the setup process in the Knocki Mobile App
```

Comment:
This now mixes pre-requisites, setting up the integration, and an example an automation.

Please split those up and restore the use of our configuration flow include/template.

Reply:
Hey @frenck! Thanks for your feedback. 

> This now mixes pre-requisites, setting up the integration, and an example an automation.
What steps would you put in `Prerequisites`? Should we remove the `What is Knocki?` section?

Thanks!

Reply:
@frenck something like this maybe?
![image](https://github.com/user-attachments/assets/4aa30e77-6195-4b1b-8200-bac960bcd916)